lua/telescope/builtin/internal.lua

```diff
@@ -951,15 +951,16 @@ internal.jumplist = function(opts)
   -- reverse the list
   local sorted_jumplist = {}
   for i = #jumplist, 1, -1 do
-    jumplist[i].text = ''
+    jumplist[i].text = vim.api.nvim_buf_get_lines(jumplist[i].bufnr, jumplist[i].lnum, jumplist[i].lnum+1,
+      false)[1] or ''
     table.insert(sorted_jumplist, jumplist[i])
```

Comment:
Hmm maybe we should only do it if the buffer is valid. before that we had the check in entry maker but now we should do it before we do `nvim_buf_get_lines` otherwise it throws an error
So lets only do it `vim.api.nvim_buf_is_valid(jumplist[i].bufnr` just in case

Reply:
So apparently `:jumps` only outputs jumplist entries with valid buffers, while `vim.fn.getjumplist()` will output entries with invalid buffers.

Based on your suggestion, we should more closely emulate `:jumps`, which I agree with. Pushed the change.